src/Symfony/Component/HttpClient/Exception/HttpExceptionTrait.php

```diff
@@ -20,8 +20,12 @@
  */
 trait HttpExceptionTrait
 {
+    /** @var ResponseInterface */
```

Comment:
should be removed

Reply:
the solution with a new interface looks cleaner, I'll do this way if there are no objections